app/i18n/raw-i18n/translations/es.json

```diff
@@ -524,7 +524,7 @@
         "paycode": "Código de pago",
         "onchain": "Onchain",
         "bitcoin": "Bitcoin",
-        "stablesats": "Stablesats\n ",
+        "stablesats": "Stablesats",
```

Comment:
this may need to be changed in transifex? (not sure if the sync goes both way, we'll see)